test/GraphTest.cpp

```diff
@@ -105,6 +105,36 @@ TEST(GraphTest, GetNodeSet_2) {
               }) != nodeSet.end());
 }
 
+TEST(GraphTest, FindEdge_Test) {
+  CXXGraph::Node<int> node1("1", 1);
+  CXXGraph::Node<int> node2("2", 2);
+  CXXGraph::Node<int> node3("3", 3);
+  CXXGraph::Node<int> node4("4", 2);
+  CXXGraph::Node<int> node5("5", 3);
+  std::pair<const CXXGraph::Node<int> *, const CXXGraph::Node<int> *> pairNode(
```

Comment:
`pairNode` is defined but never used, so you can delete it.
This will solve one of the failed workflows.

Reply:
Done. Please review.